### PR TITLE
Update how to set your Git credentials to reference new Step

### PR DESCRIPTION
**Pull request by dphillipsprosper**
I figure this is the last piece of the puzzle in creating the step and submitting it to the public step repo - update the documentation to point to it.

### CODE REVIEW

**Comment by dphillipsprosper** [comment]
@zoltan-baba - I see you review these from the closed PR's. Is there anything else I need to do?

File: _articles/en/builds/setting-your-git-credentials-on-build-machines.md [hunk 1]
```diff
@@ -19,17 +19,15 @@ The default Git username and user email address on our Stacks are the following:
 
 If you want to push back  (`git push`) any commits to your own repo from Bitrise **while running your build**, you have to **set your own username and the email address**. Here is how to do it!
```
Comment (zoltan-baba) [inline _articles/en/builds/setting-your-git-credentials-on-build-machines.md]:
```suggestion
If you want to push back  (`git push`) any commits to your own repo from Bitrise **while running your build**, you have to **set your own username and the email address**. There are two ways to achieve this:

- You can use a custom Script Step to set your credentials with the `git config` command.
- You can use the **Set Git Credentials** Step. 

Let's go through the latter option in detail:
```

**Comment by zoltan-baba** [comment]
Hi! The Step is now in the public steplib repository: https://github.com/bitrise-io/bitrise-steplib/tree/master/steps/set-git-credentials. And thank you very much for your work! I've added a small suggestion to make sure users are aware that they can use the Script Step, if they prefer that for whatever reason. 

For future reference: we'll soon be posting an updated version of our Step development guidelines. So in the future our contributors will have a much more detailed guide to follow when developing Steps.

Thanks again!

**Comment by dphillipsprosper** [comment]
@zoltan-baba - I committed your suggestion. From my end, everything looks good.

**Comment by zoltan-baba** [comment]
Thanks @dphillipsprosper! I've made a note to let you know when the new guidelines are out!

**Pull request merged**